src/components/views/voip/VideoFeed.js

```diff
@@ -21,9 +21,23 @@ var React = require('react');
 module.exports = React.createClass({
     displayName: 'VideoFeed',
 
+    componentDidMount() {
+        this.refs.vid.addEventListener('resize', this.onResize);
+    },
+
+    componentWillUnmount() {
+        this.refs.vid.removeEventListener('resize', this.onResize);
+    },
+
+    onResize: function(e) {
+        if(this.props.onResize) {
+            this.props.onResize(e);
```

Comment:
It's also worth pointing out you can be super lazy and do this instead:

```javascript
getDefaultProps: {
  return {
    onResize: function(e) {}
  };
},

componentDidMount: function() {
  this.refs.vid.addEventListener('resize', this.props.onResize);
},

...
```

which means you can remove this proxying function entirely.